src/app/shared/context.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from '@angular/core';
+import { Context } from './../models/context';
+import { DummyService } from './../dummy/dummy.service';
+import { Router } from '@angular/router';
```

Comment:
This is a style thing but it is worth noting. We usually group the '@angular' imports first and then after a space have the local imports. If there are a lot of imports we tend to group them further based on the folder structure.